CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# CHANGELOG for `parameter_substitution`
+
+Inspired by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+Note: This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [1.0.0] - 2020-05-18
+### Added
+- Replace hobo_support with invoca-utils
+
+[0.3.0]: https://github.com/Invoca/parameter_substitution/compare/v0.1.0...v1.0.0
```

Comment:
This should be `[1.0.0]` right? And looks like it's starting from `v0.2.3`.
```suggestion
[1.0.0]: https://github.com/Invoca/parameter_substitution/compare/v0.2.3...v1.0.0
```

Reply:
Good catch. Updated.